src/main.rs

```diff
@@ -374,13 +377,16 @@ async fn main() -> Result<()> {
 /// * Returns an error if network slug is missing when block number is specified
 /// * Returns an error if monitor execution fails for any reason (invalid path, network issues, etc.)
 #[instrument(skip_all)]
+#[allow(clippy::too_many_arguments)]
 async fn test_monitor_execution(
 	path: String,
 	network_slug: Option<String>,
 	block_number: Option<u64>,
 	monitor_service: Arc<Mutex<MonitorServiceType>>,
 	network_service: Arc<Mutex<NetworkService<NetworkRepository>>>,
 	filter_service: Arc<FilterService>,
+	trigger_execution_service: Arc<TriggerExecutionService<TriggerRepository>>,
+	active_monitors_trigger_scripts: HashMap<String, (ScriptLanguage, String)>,
 	raw_output: bool,
```

Comment:
What do you think about creating a new type `MonitorExecutionTestConfig` to encapsulate these params instead?